LocalSettings.php

```diff
@@ -4333,6 +4333,25 @@
 	'wgMediaViewerEnableByDefault' => [
 		'default' => true,
 	],
+	'wgMediaViewerThumbnailBucketSizes' => [
+		'default' => [
+			320,
+			800,
+			1024,
+			1280,
+			1920,
+			2560,
+			2880
```

Comment:
Slight nitpick: We usually have trailing commas at the end of (multiline) arrays :p

Reply:
This has been addressed, though your update on Phorge seems to suggest that the root cause is some other configuration problem, so this change may not be necessary.